### ISSUE TITLE
Request for the Help for the Clarification on Environment Versions

### ISSUE THREAD

**Issue by StaminaTang** [KaiYan289/RL_as_Vitamin_for_Online_Decision_Transformers#1]
Hello!
I hope you're doing well!

I would greatly appreciate it if you could kindly provide more details about the environments used, such as the versions of stable_baselines3, gym_robotics, and any other related dependencies (is shimmy or gymnasium used here? gym_robotics or gymnasium_robotics?).

From the testing, when using the versions recommended in the readme.md (e.g., gym, d4rl, etc.), I noticed that with stable_baselines3 versions 1.6.2 or 2.0.0a, shimmy==1.3.0, the performance in the AntMaze environments appears as follows:
<img width="834" alt="image" src="https://github.com/user-attachments/assets/c4c0874d-7c13-4d8a-80db-9bd335fe02bc">

The expected performance should be above 0 and closer to 100, but in the current environment, the obtained performance is consistently below zero for typical antmaze tasks. This seems to be related to the version control, and it might also be affecting the Maze environment.

2. Maybe it is related to the commands I used?
I am not sure whether the following command I use is correct, and could you please tell me the variable refers to Pretrain α:
<img width="700" alt="image" src="https://github.com/user-attachments/assets/d6474b00-f7f6-4f71-91e1-96a65d921b04">

`python main.py --env antmaze-umaze-v2 --eval_rtg -100 --online_rtg -100 --weight_decay 0.0001 --rl_algo TD3 --num_actor_update_interval 1 --actor_rl_coeff 0.1 --use_entropy_reg 0 --stoc 0 --actor_sup_coeff 1 --minimum_sapairs_per_iter 1000 --num_updates_per_pretrain_iter 40000 --max_pretrain_iters 40 --eval_context_length 1 --K 5 --critic_learning_rate 0.0002 --seed 16 --max_online_iters 99999 --replay_size 2000 --RL_from_start 1 --gamma 0.998`

3. Or do we have to change some parts of the codebase when running the different environments?

4. Besides, is the final result derived from evaluation/return_mean_gm? Is the before result derived from it too?

Thank you very much for your help!!! 
Have a nice day!!!

**Comment by KaiYan289** [comment]
Hi StaminaTang,

Thanks for your attention for our work! For antmaze environments, following CQL and IQL, we have conducted reward shaping which is to apply a -1 reward for every timestep; therefore, the original reward being 1 is equivalent to number of steps < 1000 (i.e. return_mean_gm > -1000). The original reward is reflected in the SR (Success Rate) part of the wandb log file. This is mentioned in our Appendix F.1.3.

**Comment by StaminaTang** [comment]
Thank you very much for your quick reply!!! 

Do you think the command above is correct and could you please tell me which variable refers to Pretrain α. 

Could you please tell me the versions of stable_baselines3 and gym_robotics?

**Comment by KaiYan289** [comment]
--actor_rl_coeff 0.1 refers to pretrain alpha (which is the same as finetuning alpha). It will only be enabled when you set --RL_from_start 1; otherwise it is 0. We use stable-baselines3==1.1.0, gym==0.23.1 under python 3.8.5.

**Comment by StaminaTang** [comment]
Thank you very much!
